swagger2.0/oic.sec.ael.swagger.json

```diff
@@ -0,0 +1,263 @@
+{
+	"swagger": "2.0",
+	"info": {
+		"title": "Security Event List",
```

Comment:
The title of the Resource is "Auditable Event List"

Reply:
Fixed in https://github.com/openconnectivityfoundation/security/pull/120